eclair-core/src/main/resources/reference.conf

```
@@ -29,6 +29,7 @@ eclair {
     wallet = ""
     zmqblock = "tcp://127.0.0.1:29000"
     zmqtx = "tcp://127.0.0.1:29000"
+    batching = true
```

Maybe rename and document:

```suggestion
    // Batching requests saves bandwidth but may slightly degrade latency and reliability.
    // You may want to disable this when bitcoin is running on a remote machine with an unreliable network.
    batch-requests = true
```